packages/p2p-v2/src/utils/time.ts

```diff
@@ -58,3 +58,81 @@ export const formatTime = (minutes: number) => {
 
     return `${hours} ${hoursText} ${remainingMinutes} ${minutesText}`;
 };
+
+/**
+ * Gets the formatted date string in the format "DD MMM YYYY HH:mm:ss". e.g.: "01 Jan 1970 21:01:11"
+ * or "MMM DD YYYY HH:mm:ss" for local time. e.g.: "Jan 01 1970 21:01:11" or without seconds if
+ * hasSeconds is false. e.g.: "01 Jan 1970 21:01" or "Jan 01 1970 21:01".
+ *
+ * @param {Date} dateObj - The date object to format.
+ * @param {boolean} isLocal - Whether to use local time or UTC time.
+ * @param {boolean} hasSeconds - Whether to include seconds in the time.
+ * @returns {String} The formatted date string.
+ */
+export const getFormattedDateString = (
```

Comment:
Raise this file up to the @deriv-com/utils team as this is one of the shared items.